### PR TITLE
Added sphinx docs

### PR DESCRIPTION
**Pull request by alijared**
In-depth docs for Croud now reside within the Croud repository

### CODE REVIEW

File: DEVELOP.rst [hunk 1]
```diff
@@ -74,3 +74,28 @@ See the instructions for `Generating distribution archives`_ for more details.
...
 .. _PyPI: https://pypi.org/project/croud/
 .. _Test PyPI: https://packaging.python.org/guides/using-testpypi/
+
+
+Writing Documentation
+=====================
+
+The docs live under the ``docs`` directory.
+
+The docs are written written with ReStructuredText_ and processed with Sphinx_.
```
Comment (christianbader) [inline DEVELOP.rst]: remove one written

File: docs/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# don't pin crate version numbers so the latest will always be pulled when you
```
Comment (christianbader) [inline docs/requirements.txt]: this can be removed, i guess

File: docs/docs/getting-started.txt [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+To install using pip, simply run the following command:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+.. _git:
+
+Git
```
Comment (lukas-bernert) [inline docs/docs/getting-started.txt]: Can we please move the Git section to the developer docs?

File: docs/docs/commands.txt [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+    Most commands are split by the resource that they interact with.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+.. _login:
+
+``login``
```
Comment (lukas-bernert) [inline docs/docs/commands.txt]: As we've added a "options" table to the commands below I think we should also mention that the user can specify the environment at login.

File: README.rst [hunk 5]
```diff
@@ -14,17 +14,10 @@ Using pip
...
 
-    $ git clone git@github.com:crate/croud.git && cd croud/
-    $ python3.6 -m venv env
-    $ env/bin/pip install -e .
-    $ export PATH=$PATH:$(pwd)/env/bin/croud
+To update, run::
 
+    $ pip install -U croud
 
 Usage
```
[the remark this answers is not in the document]
Comment (alijared) [reply README.rst]: Crash has how to update in the README as well. Do you still want me to remove it?

**Pull request merged**